platform.md

```diff
@@ -840,6 +840,8 @@ Usage:
       - `run-image.reference` SHALL uniquely identify `<run-image>`
       - `run-image.top-layer` SHALL be set to the uncompressed digest of the top layer in `<run-image>`
     - The value of `io.buildpacks.stack.*` labels SHALL be modified to that of the new `run-image`
+      - `stack.run-image.image` SHALL uniquely identify `<run-image>`
+      - `stack.run-image.mirrors` SHALL be updated to include the new image mirrors
```

Comment:
How will the run image mirrors be identified?

Reply:
(During build they come from stack.toml - soon to be run.toml in future platform APIs - but we don't have this file during rebase.)

Reply:
Changes has been made @natalieparellano